src/Features/Core/Portable/SolutionCrawler/SolutionCrawlerRegistrationService.cs

```diff
@@ -44,6 +44,22 @@ internal partial class SolutionCrawlerRegistrationService : ISolutionCrawlerRegi
         }
 
         public void Register(Workspace workspace)
+        {
+            EnsureRegistration(workspace, initializeLazily: true);
+        }
+
+        /// <summary>
+        /// make sure solution cralwer is registered for the given workspace.
+        /// 
+        /// we need this until https://github.com/dotnet/roslyn/issues/35514 is fixed.
```

Comment:
Confused -- I thought that was being fixed by this PR?

Reply:
removed.